includes/Plugin.php

```diff
@@ -317,4 +319,15 @@ public function register_rest_routes() {
 		$stories_settings = new Stories_Settings_Controller();
 		$stories_settings->register_routes();
 	}
+
+	/**
+	 * Register Widgets.
+	 *
+	 * @return void
+	 */
+	public function register_widgets() {
+		if ( current_theme_supports( 'web-stories' ) ) {
```

Comment:
I would also like to understand this. Wouldn't that seriously limit the number of people that even see this feature? We don't limit that embed block by theme support, I don't see why we should limit the widget here. A user uses the widget by choice and if it is broken, it is pretty simple fix to remove the widget.

Reply:
Initially I added widget based on theme support, but after discussion we decided to remove theme support conditions. This is a place I missed to remove. Removed it now. 
Agree with idea that this will limit the beneficiaries significantly.